libs/openFrameworks/types/ofBaseTypes.cpp

```diff
@@ -47,7 +47,24 @@ ofBaseVideoPlayer::~ofBaseVideoPlayer(){
 
 }
 
+//---------------------------------------------------------------------------
+bool ofBaseVideoPlayer::load(const of::filesystem::path & fileName){
+	return load(fileName.string());
+}
+
+//---------------------------------------------------------------------------
+bool ofBaseVideoPlayer::load(const std::string & fileName){
+	ofLogError("ofBaseVideoPlayer") << " One of the two load functions need to be implemented ";
+	return false;
+}
+
+//---------------------------------------------------------------------------
 void ofBaseVideoPlayer::loadAsync(const of::filesystem::path & fileName){
+	loadAsync(fileName.string());
```

Comment:
we can use ```ofPathToString(fileName)```  here

Reply:
Indeed. the string() function can throw exceptions and because of that the ofPathToString handles it. In the switch to use fs::path we should enforce the use of ofPathToString rather than calling string() directly.

Reply:
ahh good catch! thanks guys!